RELEASES.md

```diff
@@ -79,4 +91,85 @@ deadline of 3 weeks.
+  they are fixed or bump them to the next milestone.
+* Begin marshalling the ongoing PR flow in this repo. Ask maintainers to hold off merging any
+  particularly risky PRs until after the release is tagged. This is because we aim for main to be
+  at release candidate quality at all times.
+* Do a final check of the [release notes](docs/root/version_history/current.rst):
+  * Make any needed corrections (grammar, punctuation, formatting, etc.).
+  * Check to see if any security/stable version release notes are duplicated in
+    the major version release notes. These should not be duplicated.
+  * If the "Deprecated" section is empty, delete it.
+  * Remove the "Pending" tags and add dates to the top of the [release notes for this version](docs/root/version_history/current.rst).
+  * Switch the [VERSION.txt](VERSION.txt) from a "dev" variant to a final variant. E.g., "1.6.0-dev" to
+    "1.6.0".
+  * Update the [RELEASES](RELEASES.md) doc with the relevant dates. Now, or after you cut the
+    release, please also make sure there's a stable maintainer signed up for next quarter,
+    and the deadline for the next release is documented in the release schedule.
+  * Get a review and merge.
+* Wait for tests to pass on [main](https://dev.azure.com/cncf/envoy/_build).
+* Create a [tagged release](https://github.com/envoyproxy/envoy/releases). The release should
+  start with "v" and be followed by the version number. E.g., "v1.6.0". **This must match the
+  [VERSION](VERSION).**
+* From the envoy [landing page](https://github.com/envoyproxy/envoy) use the branch drop-down to create a branch
+  from the tagged release, e.g. "release/v1.6". It will be used for the
+  [stable releases](RELEASES.md#stable-releases).
+* Tagging will kick off another run of [AZP postsubmit](https://dev.azure.com/cncf/envoy/_build?definitionId=11). Monitor that
+  tag build to make sure that the final docker images get pushed along with
+  the final docs. The final documentation will end up in the
+  [envoy-website repository](https://github.com/envoyproxy/envoy-website/tree/main/docs/envoy).
+* Update the website ([example PR](https://github.com/envoyproxy/envoy-website/pull/148)) for the new release.
+* Craft a witty/uplifting email and send it to all the email aliases including envoy-announce@.
+* Make sure we tweet the new release: either have Matt do it or email social@cncf.io and ask them to do an Envoy account
+  post.
+* Do a new PR to setup the next version
+  * Update [VERSION.txt](VERSION.txt) to the next development release. E.g., "1.7.0-dev".
+  * `git mv docs/root/version_history/current.rst docs/root/version_history/v1.6.0.rst`, filling in the previous
+    release version number in the filename and delete empty sections (like Incompatible Behavior Changes, Minor Bahavior Changes, etc).
+    Add an entry for the new file in the `toctree` in
+    [version_history.rst](docs/root/version_history/version_history.rst).
```

Comment:
```markdown
* Edit the file you just created (eg `docs/root/version_history/v1.6.0.rst`) replacing the link part (between the `<>`) of any `ref:` links to point at the version - eg `` :ref:`Some link text <actual link>` `` -> `` :ref:`Some link text <v1.16:actual link>` ``

Reply:
Thanks! Added!

Reply:
im noticing that the versions are inconsistent in both the original and my suggestion - im guessing we should be using `1.16` everywhere except for the instances of `1.17` (or update both)

Reply:
my apologies - seems like it was just my suggestion that introduced `1.16` - stilll, i think would probably good to make sure this is consistent

Reply:
Hm. I think we're using 1.6.0 and 1.7.0 in the examples so I changed the 1.16 to 1.6.0 which I think is correct. Work for you?

Reply:
yep, for sure